node/request.js

```diff
@@ -91,7 +92,13 @@ TChannelRequest.prototype.emitError = function emitError(err) {
     TChannelOutRequest.prototype.emitLatency.call(self);
 
     self.channel.services.onRequestError(self);
-    self.errorEvent.emit(self, err);
+
+    if (LiteError.isLiteError(err)) {
```

Comment:
This is not a good idea. Far too generic. Let's just start propagating errorFrame events from V2Handler and have Connection and Request handle things properly